dashboard/src/components/InfoCard/InfoCard.tsx

```diff
@@ -10,46 +10,50 @@ export interface IServiceInstanceCardProps {
   info: string;
   link?: string;
   icon?: string;
+  description?: string | JSX.Element;
   tag1Class?: string;
   tag1Content?: string | JSX.Element;
   tag2Class?: string;
   tag2Content?: string | JSX.Element;
 }
 
 const InfoCard: React.SFC<IServiceInstanceCardProps> = props => {
-  const { title, link, info, tag1Content, tag1Class, tag2Content, tag2Class } = props;
+  const { title, link, info, description, tag1Content, tag1Class, tag2Content, tag2Class } = props;
   const icon = props.icon ? props.icon : placeholder;
   return (
     <Card responsive={true} className="ListItem">
       <Link to={link || "#"} title={title}>
         <CardIcon icon={icon} />
-        <CardContent>
-          <div className="ListItem__content">
+      </Link>
+      <CardContent>
+        <div className="ListItem__content">
+          <Link to={link || "#"} title={title}>
             <h3 className="ListItem__content__title type-big">{title}</h3>
-            <div className="ListItem__content__info">
-              <p className="margin-reset type-small padding-t-tiny type-color-light-blue">{info}</p>
-              <div>
-                {tag1Content && (
-                  <span
-                    className={`ListItem__content__info_tag ListItem__content__info_tag-1 type-small type-color-white padding-t-tiny padding-h-normal ${tag1Class ||
-                      ""}`}
-                  >
-                    {tag1Content}
-                  </span>
-                )}
-                {tag2Content && (
-                  <span
-                    className={`ListItem__content__info_tag ListItem__content__info_tag-2 type-small type-color-white padding-t-tiny padding-h-normal margin-b-normal ${tag2Class ||
-                      ""}`}
-                  >
-                    {tag2Content}
-                  </span>
-                )}
-              </div>
+          </Link>
+          {description}
+          <div className="ListItem__content__info">
+            <p className="margin-reset type-small padding-t-tiny type-color-light-blue">{info}</p>
+            <div>
+              {tag1Content && (
+                <span
+                  className={`ListItem__content__info_tag ListItem__content__info_tag-1 type-small type-color-white padding-t-tiny padding-h-normal ${tag1Class ||
+                    ""}`}
+                >
+                  {tag1Content}
+                </span>
+              )}
+              {tag2Content && (
+                <span
+                  className={`ListItem__content__info_tag ListItem__content__info_tag-2 type-small type-color-white padding-t-tiny padding-h-normal margin-b-normal ${tag2Class ||
+                    ""}`}
+                >
+                  {tag2Content}
+                </span>
+              )}
             </div>
           </div>
-        </CardContent>
-      </Link>
```

Comment:
Actually like this, I think it makes sense that the whole content is linkable. Why did you change it?

Reply:
_replied before_